extensions/positron-python/python_files/posit/positron/utils.py

```diff
@@ -71,6 +71,11 @@ def get_qualname(value: Any) -> str:
         # Finally, try to return the generic type's name, otherwise report object
         qualname = getattr(type(value), "__name__", "object")
 
+    # In the rare situation an object incorrectly handles __qualname__ by not returning
+    # a str, we fall back to the name of the type
+    if type(qualname) is not str:
```

Comment:
```suggestion
    if not isinstance(qualname, str):
```

nit, more pythonic way to do this